packages/cli/src/commands/setup/ui/libraries/tailwindcss.js

```diff
@@ -151,6 +156,36 @@ export const handler = async ({ force, install }) => {
         }
       },
     },
+    {
+      title: 'Adding recommended VS Code extensions...',
+      task: (_ctx, task) => {
+        const VS_CODE_EXTENSIONS_PATH = path.join(
+          rwPaths.base,
+          '.vscode/extensions.json'
+        )
+
+        if (!usingVSCode()) {
+          task.skip("Looks like your're not using VS Code")
+        } else {
+          let originalExtensionsJson = { recommendations: [] }
+          if (fs.existsSync(VS_CODE_EXTENSIONS_PATH)) {
+            const originalExtensionsFile = fs.readFileSync(
+              VS_CODE_EXTENSIONS_PATH,
+              'utf-8'
+            )
+            originalExtensionsJson = JSON.parse(originalExtensionsFile)
+          }
+          const newExtensionsJson = {
+            ...originalExtensionsJson,
+            recommendations: [
+              ...originalExtensionsJson.recommendations,
+              ...recommendedVSCodeExtensions,
+            ],
+          }
+          fs.writeFileSync(VS_CODE_EXTENSIONS_PATH, newExtensionsJson)
```

Comment:
Oops - so I misread this a little bit.
```suggestion
          fs.writeFileSync(VS_CODE_EXTENSIONS_PATH, JSON.stringify(newExtensionsJson, null, 2))
```

This works well, and formats the JSON nicely

Reply:
@zygopleural still needs this :). fsWriteFileSync needs a string, we were passing a JS object!

Reply:
Done @dac09